js/components/portal-dashboard/response-details/popup-class-nav.tsx

```diff
@@ -14,28 +14,20 @@ import cssClassNav from "../../../../css/portal-dashboard/class-nav.less";
 interface IProps {
   anonymous: boolean;
   isSpotlightOn: boolean;
+  inQuestionMode: boolean;
   onShowDialog: (show: boolean) => void;
   questionCount: number;
   setAnonymous: (value: boolean) => void;
   setStudentSort: (value: string) => void;
   sortByMethod: string;
   studentCount: number;
+  setListViewMode: (value: boolean) => void;
   trackEvent: (category: string, action: string, label: string) => void;
 }
-interface IState {
-  inQuestionMode: boolean;
-}
-export class PopupClassNav extends React.PureComponent<IProps, IState>{
-  constructor(props: IProps) {
-    super(props);
-    this.state = {
-      inQuestionMode: false
-    };
-  }
```

Comment:
With this now stateless, we probably want to consider making it a functional component.  Not needed now, however.  Just thinking out loud.